styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```

Comment:
- [ OPTIONAL]You should use a class selector as required for this  A .css file that includes one class with styles for your header (make it in your favorite color). Kindly use a class selector for semantically reason :wink: